sdk/maps/Azure.Maps.Search/src/MapsSearchClient.cs

```diff
@@ -121,768 +118,71 @@ public MapsSearchClient(AzureSasCredential credential, MapsSearchClientOptions o
-                );
-            }
-            catch (Exception e)
-            {
-                scope.Failed(e);
-                throw;
-            }
-        }
-
-        /// <summary>
-        /// Get POI by Category API. Points of Interest (POI) Category Search allows you to request POI results from given category. Search allows to query POIs from one category at a time. Endpoint will only return POI results which are categorized as specified.  Response includes POI details such as address, a pair of coordinates location and classification.
-        /// </summary>
-        /// <param name="options"> additional options  </param>
-        /// <param name="cancellationToken"> The cancellation token to use. </param>
-        public virtual async Task<Response<SearchAddressResult>> SearchPointOfInterestCategoryAsync(SearchPointOfInterestCategoryOptions options = null, CancellationToken cancellationToken = default)
-        {
-            using var scope = _clientDiagnostics.CreateScope("MapsSearchClient.SearchPointOfInterestCategory");
-            scope.Start();
-            try
-            {
-                LocalizedMapView? localizedMapView = null;
-                if (options?.LocalizedMapView != null)
-                {
-                    localizedMapView = new LocalizedMapView(options?.LocalizedMapView.ToString());
-                }
-                return await RestClient.SearchPointOfInterestCategoryAsync(ResponseFormat.Json, options?.query, options?.IsTypeAhead, options?.Top, options?.Skip, options?.CategoryFilter, options.CountryFilter, options?.Coordinates?.Latitude, options?.Coordinates?.Longitude, options?.RadiusInMeters, options?.BoundingBox != null ? options.BoundingBox.North + "," + options.BoundingBox.West : null, options?.BoundingBox != null ? options.BoundingBox.South + "," + options.BoundingBox.East : null, options?.Language.ToString(), options?.ExtendedPostalCodesFor, options?.BrandFilter, options?.ElectricVehicleConnectorFilter, localizedMapView, options?.OperatingHours, cancellationToken).ConfigureAwait(false);
-            }
-            catch (Exception e)
-            {
-                scope.Failed(e);
-                throw;
-            }
-        }
-
         /// <summary>
-        /// Get POI by Category API. Points of Interest (POI) Category Search allows you to request POI results from given category. Search allows to query POIs from one category at a time. Endpoint will only return POI results which are categorized as specified.  Response includes POI details such as address, a pair of coordinates location and classification.
+        /// **Geocoding**
+        ///
+        /// **Applies to:** see [pricing tiers](https://aka.ms/AzureMapsPricingTier).
+        ///
```

Comment:
Please help to remove all the "pricing tiers" description. Like L#124-127, L#195-198, L#302-305, etc.

Reply:
@dubiety done